index.js

```diff
@@ -131,6 +131,41 @@ Limiter.prototype.get = function (fn) {
 };
 
 /**
+ * reset rate limit
+ *
+ * @api public
+ */
+Limiter.prototype.reset = function(fn){
+  
+  var count = this.prefix + 'count';
+  var limit = this.prefix + 'limit';
+  var reset = this.prefix + 'reset';
+  var duration = this.duration;
+  var max = this.max;
+  var db = this.db;
+  
+  function recreate() {
```

Comment:
reusing `create` function inside `Limiter.prototype.get` will increase scope of changes and will effect whole package so i try making only minimum changes without effecting other parts.

changes if we go with reuse `create` function
 - new version has callback parameter `fn` as `mget` and `decr` too
 - moving `create`, `mget` and `decr` outside of  `Limiter.prototype.get`

so is this acceptable if so, i can update this pull request with these changes